asciinema2gif

```diff
@@ -118,3 +118,5 @@ convert -delay 5 -loop 0 frames/*.png gif:- | gifsicle --colors=256 --delay=6 --
 cd - >/dev/null
 mv "${tempdir}/asciicast.gif" "${output_file}"
 echo ">> Generated in: ${output_file}"
+echo "cleaning up the frames before quiting"
+find "${tempdir}" -name \*.png -delete
```

Comment:
The temporary directory is created with `mktemp` [on line 10](https://github.com/jfmherokiller/asciinema2gif/blob/cf438979512b6c424468094d58867b79c9af8921/asciinema2gif#L10), so it will be unique. This means there’s no point in using `find`, and you can just remove the whole directory (`rm -rf "${tempdir}"`) as it’ll only have temporary files related to that particular run of this particular script.